src/Tools/SarifValidator/SarifValidator.csproj

```diff
@@ -23,18 +23,6 @@
       <HintPath>..\..\packages\CommandLineParser.2.0.275-beta\lib\net45\CommandLine.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.CSS.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.CodeAnalysis.SarifValidationLibraries.1.0.0\lib\net46\Microsoft.CSS.Core.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Microsoft.JSON.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.CodeAnalysis.SarifValidationLibraries.1.0.0\lib\net46\Microsoft.JSON.Core.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="Microsoft.Web.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.CodeAnalysis.SarifValidationLibraries.1.0.0\lib\net46\Microsoft.Web.Core.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
```

Comment:
The SarifValidator tool now depends on the Sarif.Validation project (which in turn depends on the VS assemblies package), rather than depending directly on those assemblies.